internal/daemon/vendor-specific-plugins/marvell/config.yaml

```diff
@@ -0,0 +1,2 @@
+portType: "veth"  # Currently supported values are "veth","hwlbk"     
```

Comment:
Can we hardcode these values for now? (instead of having a config file, or is this necessary for debugging?)

The proper way is to create a CR or config map from the Operator side. But we need a proper proposal of what is needed, I don't think we are ready for this yet.